examples/opt_serving/generator.py

```diff
@@ -137,6 +137,50 @@ def load_model(self):
             self.num_gpus = alpa.get_global_cluster().num_devices
         logger.info(f"Loading model time: {load_time:.2f}")
 
+    # FIXME(Hao): the current batch uses the topp, reponse_length of the first sentence.
+    def pad_batch(self, inputs, pad_value=None):
+        if pad_value is None:
+            pad_value = self.pad
+        new_inputs = inputs
+        src_lens = [len(input) for input in inputs]
+        max_len = max(src_lens)
+        bs = len(inputs)
+
+        for new_input in new_inputs:
+            ori_len = len(new_input)
+            if len(new_input) < max_len:
+                new_input.extend([pad_value for _ in range(max_len - ori_len)])
+        # pad to bs = MAX_BS
+        if bs < MAX_BS:
+            new_inputs.extend([[pad_value for _ in range(max_len)] for _ in range(MAX_BS - bs)])
+        return new_inputs
+    
+    def __call__(
```

Comment:
I used `__call__` because it's essentially just wrapping the OPT model's forward function, which would normally be accessed by just doing e.g. `model(**args)`. I could rename it to `forward` instead and explicitly call `.forward()` if that would be preferred? What do you think?

Reply:
SGTM